src/menu_admin.c

```diff
@@ -22,7 +26,7 @@ void menu_admin(int user) {
     /* clear the console window */
     char choice;
 
-    user = 1000000;
+    user = 1000000; /* Admin user */

```

Comment:
Do not use hard coded value

Reply:
the admin user value has to be hard coded because the admin has all the rights to create employees so as to create a database of employees there should be admin that has the access to do so.

Reply:
ok.